gradle.properties

```diff
@@ -1,19 +1,19 @@
-# Project-wide Gradle settings.
-
-# IDE (e.g. Android Studio) users:
-# Gradle settings configured through the IDE *will override*
-# any settings specified in this file.
-
+## Project-wide Gradle settings.
```

Comment:
What are the reasons for the changes to this file? Did you copy it from somewhere or what do you want to achieve with tme?